src/management/cameras.php

```diff
@@ -0,0 +1,109 @@
+<?php
+$servername = "db";
+$username = "db_username";
+$password = "db_password";
+$dbname = "db_name";
```

Comment:
DB config is already added in the `index.php` file. I recommend moving it to a separate file ex. `config.php`.